docker-compose.yml

```diff
@@ -8,8 +44,19 @@ services:
     ports:
       - "4242:4242"
     environment:
+      # This points Unleash to its backing database (defined in the `db` section below)
       DATABASE_URL: "postgres://postgres:unleash@db/postgres"
+      # Disable SSL for database connections. @chriswk: why do we do this?
       DATABASE_SSL: "false"
```

Comment:
@chriswk Do you want to explain to me again what this does and why it's necessary?

Reply:
Because otherwise we'd have to configure an SSL certificate for the postgres container, and get the unleash image to trust a custom SSL certificate. Instead we run an unencrypted connection against postgres in this demo setup